vacasa/connect/connect.py

```diff
@@ -1165,6 +1167,8 @@ def create_contact(self,
             phone: Phone Number of contact
             phone_notes: Corresponds to “Phone Notes” on Contact page in Admin
             language_id: Foreign key to table languages
+            created_by: Who created the contact
```

Comment:
Is this the person's first name, full name, or some other sort of identifier?

Reply:
updated comment to be more specific.

Reply:
Thanks!